articles/app-service/overview-inbound-outbound-ips.md

```diff
@@ -80,6 +80,8 @@ az webapp show --resource-group <group_name> --name <app_name> --query possibleO
 (Get-AzWebApp -ResourceGroup <group_name> -name <app_name>).PossibleOutboundIpAddresses
 ```
 
+For Function Apps See [Function App outbound IP addresses](https://learn.microsoft.com/en-us/azure/azure-functions/ip-addresses?tabs=azure-powershell#find-outbound-ip-addresses)
```

Comment:
Hi @rmarr2,

update the link in the parenthesis to:
(../azure-functions/ip-addresses?tabs=azure-powershell#find-outbound-ip-addresses)

Tag me once done and I'll approve.